src/auth/authController.ts

```diff
@@ -56,6 +57,53 @@ export class AuthController {
         ctx.body = err.message;
         return;
       }
+      else if( err instanceof EmailNotVerifies){
+        ctx.status = 400;
+        ctx.body = err.message;
+        return;
+      }
```

Comment:
after else if you need to add
`throw err;`
because if there will be not an WrongCredentials or EmailNotVerifies error you will not know about it